### PR TITLE
Update decimal encoding docs

### PR DESCRIPTION
**Pull request by fayi-da**
CHANGELOG_BEGIN
- [DOCS] Update docs for Decimal encoding CHANGELOG_END

### CODE REVIEW

File: docs/source/json-api/lf-value-specification.rst [hunk 1]
```diff
@@ -100,10 +100,10 @@ A few invalid examples::
...
 ======
 
-If encodeDecimalAsString is set, decimals are encoded as strings, using
-the format ``-?[0-9]{1,28}(\.[0-9]{1,10})?``. If encodeDecimalAsString
-is not set, they are encoded as JSON numbers, also using the format
-``-?[0-9]{1,28}(\.[0-9]{1,10})?``.
+If encodeDecimalAsString is set, decimals are encoded as string and if
+not set, they are encoded as JSON numbers. The output is expressed as
+a number in base 10 with a maximum of 38 digits where the decimal point
+can be anywhere within the digits except the start.
```
Comment (S11001001) [inline docs/source/json-api/lf-value-specification.rst:106]: This is, strictly speaking, false; the allowable positions of the decimal point _depend on the parameter to `Numeric`_, they are not "anywhere".

**Review by S11001001 — COMMENTED** [review]
* [ ] the whole section should talk about numeric, rather than decimal
* [ ] the given bounds above the current change are wrong, too; they should be specified as dependent on the parameter to `Numeric` as well

File: docs/source/json-api/lf-value-specification.rst (hunk 1 above)
Comment (carrielaben-da) [inline docs/source/json-api/lf-value-specification.rst:106]:
```suggestion
a number in base 10 with a maximum of 38 digits where the decimal point
is set by the parameter Numeric. The decimal point cannot be before the first digit.
```

**Comment by stefanobaghino-da** [comment]
Looks stale. Can this be closed?

**Comment by stefanobaghino-da** [comment]
I'll close this for now, feel free to reopen if necessary.

**Pull request closed without merging**